ui/src/app/controllers/views/ControllerDetails/ControllerSummary/ServicesCard/ServicesCard.tsx

```diff
@@ -1,17 +1,160 @@
+  },
+  {
+    name: ServiceName.RACKD,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+    children: [
+      {
+        name: ServiceName.HTTP,
+        types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+      },
+      {
+        name: ServiceName.TFTP,
+        types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+      },
+    ],
+  },
+  {
+    name: ServiceName.DHCPD,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.DHCPD6,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.DNS_RACK,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.NTP_RACK,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.PROXY_RACK,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.SYSLOG_RACK,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+  {
+    name: ServiceName.REVERSE_PROXY,
+    types: [NodeType.RACK_CONTROLLER, NodeType.REGION_AND_RACK_CONTROLLER],
+  },
+];
+
+const generateTree = (
+  items: ServiceItem[],
+  services: Service[],
+  nodeType: NodeType
+) => (
+  <ul
+    className="p-list-tree p-list-tree--static u-no-margin--bottom"
+    role="tree"
+  >
+    {items.reduce<ReactNode[]>((acc, item) => {
+      const service = services.find((service) => service.name === item.name);
+      const includesType = item.types.includes(nodeType);
+      if (service && includesType) {
+        acc.push(
+          <li className="p-list-tree__item" key={item.name}>
+            <ServiceStatus service={service} />
+            {item.children && generateTree(item.children, services, nodeType)}
+          </li>
+        );
+      }
+      return acc;
+    }, [])}
+  </ul>
+);
+
+const generateTitle = (services: Service[]) => {
+  const { dead, degraded, running } = services.reduce(
+    (acc, service) => {
+      if (service.status === ServiceStatusName.DEAD) {
+        acc.dead += 1;
+      } else if (service.status === ServiceStatusName.DEGRADED) {
+        acc.degraded += 1;
+      } else if (service.status === ServiceStatusName.RUNNING) {
+        acc.running += 1;
+      }
+      return acc;
+    },
+    { dead: 0, degraded: 0, running: 0 }
+  );
+  const substrings = [];
+  let iconName = "";
+
+  if (running > 0) {
+    substrings.unshift(`${running} running`);
+    iconName = "success";
+  }
+  if (degraded > 0) {
+    substrings.unshift(`${degraded} degraded`);
+    iconName = "warning";
+  }
+  if (dead > 0) {
+    substrings.unshift(`${dead} dead`);
+    iconName = "error";
+  }
+
+  return (
+    <>
+      <Icon data-testid="title-icon" name={iconName} />
+      <span className="u-nudge-right--small">{substrings.join(", ")}</span>
+    </>
```

Comment:
I've slightly changed the title from the legacy version. Now, all service states will be shown if they exist instead of only the most severe state, i.e. now the title will show "1 dead, 10 running" instead of only "1 dead". Also I'm using the `error` icon instead of `power-error` since I think using power icons is misleading. A service might be broken... but the controller could still be powered on.